src/xml/xpath_context.cr

```diff
@@ -42,6 +42,7 @@ struct XML::XPathContext
   end
 
   def register_namespace(prefix, uri)
+    prefix = prefix.lchop("xmlns:")
     LibXML.xmlXPathRegisterNs(self, prefix.to_s, uri.to_s)
```

Comment:
```suggestion
    prefix = prefix.to_s.lchop("xmlns:")
    LibXML.xmlXPathRegisterNs(self, prefix, uri.to_s)
```

Reply:
I actually think the `to_s` is not needed there at all. Let me check...